packages/gnarly-reducer-erc20/src/models/sequelize.ts

```diff
@@ -0,0 +1,34 @@
+import { makeSequelizeModels as makeGnarlyModels } from '@xlnt/gnarly-core'
+
+const sequelizeModels = (Sequelize: any, sequelize: any) => {
+  const { DataTypes } = Sequelize
+  const { Patch } = makeGnarlyModels(Sequelize, sequelize)
+
+  const ERC20Balances = sequelize.define(
+    'erc20_balances',
+    {
+      id: { type: DataTypes.STRING, primaryKey: true },
+      darAddress: { type: DataTypes.STRING },
+      owner: { type: DataTypes.STRING },
+      balance: { type: DataTypes.STRING },
```

Comment:
It _may_ be helpful to have an `unsafeBalance` column that's a DECIMAL of some arbitrary precision. It's not ideal, since storing 256 bit numbers is not something postgres is designed to do, but I know a lot of people will want to sort by balance and otherwise do numeric queries, so it'd be good to support that at some basic level. You can see something similar in [`unsafeNumber`](https://github.com/XLNT/gnarly/blob/master/packages/gnarly-reducer-block-meta/src/models/sequelize.ts#L21) from the block metadata reducer.

Reply:
1. do you know what data type would be the best?
2. do we actually need it to be a string at all?

Reply:
well, turns out the `DECIMAL` type can handle the 78 decimal points required by a 256 bit number as `DECIMAL(78, 0)` so perhaps it may not be necessary. google was not very helpfulf or best practices. I've asked twitter and will report back lol https://twitter.com/mattgcondon/status/1022645438229307392

Reply:
@nionis nobody responded on twitter, but looking at various docs, it seems like a decimal is fine. So let's make the primary datatype a DECIMAL(76, 0) and a backup `strBalance` or `balanceStr` (your preference) that holds the raw data _just in case_